src/constants/regexes.ts

```
@@ -0,0 +1,23 @@
+  /[\(\)（）][\^\-_oO•°ಠ><\\\/\|~=]{1,4}[\)\(）（]/g,
+  // Table flip, shrug patterns
+  /[┻┳┃━│┌┐└┘├┤┬┴┼]/g,
+  // Common kaomoji symbols (more selective)
+  /[╯╰╭╮︵︶ツヾシ〜]/g,
+  // Decorative symbols that are typically not content
+  /[♪♫☆★♥♡‼⁉]/g,
+  // Unicode kaomoji characters (covers ᓚᘏᗢ and similar)
+  /[\u1400-\u167F\u2600-\u26FF\u2700-\u27BF\uFE00-\uFE0F]/g,
+  // Specific problematic patterns you mentioned
+  /o\(\)o/g,
+  // Additional face-like patterns with various brackets
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)